antora-ui-camel/src/partials/footer-scripts.hbs

```diff
@@ -1,5 +1,6 @@
 <script src="{{uiRootPath}}{{asset 'js/site.js'}}"></script>
 <script async src="{{uiRootPath}}{{asset 'js/vendor/highlight.js'}}"></script>
 <script async src="{{uiRootPath}}{{asset 'js/vendor/svg4everybody.js'}}"></script>
+<script async src="https://cdn.jsdelivr.net/algoliasearch/3/algoliasearchLite.min.js"></script>
```

Comment:
I think I'd like to serve this from the same server, i.e. not rely on cdn.jsdelivr.net and have cache busting like we have for other resources.